index.html

```diff
@@ -878,7 +878,10 @@ <h4>Registration Information</h4>
                     stored in a directory. 
                     The following table lists the registration information attributes for
                     use within <a>TDs</a> that embed or reference the Discovery context. 
-                    Note that only an <a>Enriched TD</a> embeds the registration information.
+                    Note that only an <a>Enriched TD</a> embeds the registration information. 
+		    <span class="rfc2119-assertion" id="tdd-context-injection">
+			An <a>Enriched TD</a> MUST contain in its @context the URI <code>https://w3c.github.io/wot-discovery/context/discovery-context.jsonld</code>
```

Comment:
I think it's time we picked and allocated more reasonable final URLs for all of these.  Note that we should also have security assertions noting that these are cached and built into implementations, and should NOT be fetched dynamically...